scraper.py

```diff
@@ -235,16 +243,25 @@ def scrape(output_file_name:str, no_of_articles:int, category_urls:dict, time_de
 
     # specify categories to scrape
     if params.categories != "all":
-        categories = params.categories.split(",")
-        categories = {category:ALL_CATEGORIES[category] for category in categories}
+        categories = params.categories.upper().split(",")
```

Comment:
why is this necessary?

Reply:
The YAML config uses upper case for its keys but categories are likely to be passed in lower case through the command line. See `env.yml` (which will be renamed `config.yml` as you requested).